examples/docs_snippets/docs_snippets/deploying/executors/code_location_executor.py

```diff
@@ -0,0 +1,8 @@
+import dagster as dg
+
+
+@dg.definitions
+def executor() -> dg.Definitions:
+    return dg.Definitions(
+        executor = dg.multiprocess_executor
```

Comment:
nit: could be on one line and typically kwargs passing has no spaces around the ` = `